<launch>
  <node pkg="spacenav_teleop" type="spacenav_teleop" name="spacenav_teleop_node" output="screen">
    <param name="twistLinearScaler" value="2"/>
    <param name="twistRotationScaler" value="2"/>
    <param name="twistTopic" value="cmd_vel"/>
    <param name="joyTopic" value="spacenav/joy"/>
  </node>
</launch>
